src/controllers/getRecipeByIdController.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response } from "express";
+import getDatabase from "../Services/databaseConnector";
+import { userIdExists, recipeIdExists } from "../Services/validators";
+import { Connection } from "promise-mysql";
+
+interface Ingredient {id: number, name: string}
+interface Recipe {
+  id: number,
+  name: string,
+  instructions: string,
+  prep_time: number,
+  cook_time: number,
+  ingredients: [Ingredient]
```

Comment:
This isn't how you type annotate arrays in TypeScript.

Reply:
done